src/Message/PurchaseRequest.php

```diff
@@ -73,6 +73,20 @@ public function getData()
         $data = parent::getData();
         $data['capture'] = 'true';
 
+        if ($level3 = $this->getLevel3()) {
+            $data['level3'] = $level3;
```

Comment:
Omnipay already has the concept of purchase "items". The word "level3" is something specific to Stripe. Figure out how to better abstract.